nni/runtime/protocol.py

```diff
@@ -7,26 +7,12 @@
 
 from .tuner_command_channel.command_type import CommandType
 from .tuner_command_channel import legacy
-from .tuner_command_channel import shim
-
-_use_ws = False
-
-def connect_websocket(url: str):
-    global _use_ws
-    _use_ws = True
-    shim.connect(url)
 
 def send(command: CommandType, data: str) -> None:
-    if _use_ws:
-        shim.send(command, data)
-    else:
-        legacy.send(command, data)
+    legacy.send(command, data)
```

Comment:
do we have somewhere still use `protocal.send` or `protocal.receive`?

Reply:
Only in legacy UT.